### ISSUE TITLE
Review and update backup & restore page

### ISSUE THREAD

**Issue by akeller** [camunda/camunda-docs#3266]
Since Identity and Web Modeler were added, the backup & restore page may need to be rewritten or cleaned up. Identity and Web Modeler content shouldn't just be added in, it should be integrated into the overall backup & restore experience.

Additionally, we need to confirm when the breaking changes were added and if/when this admonition at the top of the page could be removed. 

> It is relevant both to content today **and** the versioned docs. The problem with this page is that it has never really been touched since it was initially written so it is focused on "cluster components" (i.e. Zeebe, Tasklist, Operate, Optimize). Web Modeler is a special case here as it is cluster-independent which is why I'd refrain from adding an admonition stating that the content is outdated but suggest restructuring the page to describe the difference between backup and restore of components that live "within" the concept of a cluster and those that do not.

> Additionally, there is still a note admonition at the top of the page stating that "this release introduces breaking changes for Operate and Tasklist, as well as Optimize" which as far as I know is **not** the case in the current version. The note has been added in February 2023 (see git blame). This also applies to the detailed backup and restore pages for the components which also have this admonition.

_Originally posted by @wollefitz in https://github.com/camunda/camunda-docs/pull/3190#discussion_r1464825034_

**Comment by akeller** [comment]
@conceptualshark @npepinpe would you agree that this could be handled with https://github.com/camunda/product-hub/issues/2299? Is there anything we can do to bridge the gap between now and 8.9? I see the following breakdown:

✅ Web Modeler docs were added. 
⏭  The admonition on the [Optimize docs](https://docs.camunda.io/docs/self-managed/operational-guides/backup-restore/optimize-backup/) should be removed and isolated to a specific version, which looks like it's happening via [this PR](https://github.com/camunda/camunda-docs/pull/4829).
🛑 We also have [this PR](https://github.com/camunda/camunda-docs/pull/4791), which has stalled

**Comment by akeller** [comment]
Marking this as `severity/mid` because of the related epic planned for 8.9.

**Comment by npepinpe** [comment]
Yes, I think this will be a deliverable for that epic. It's likely that InfraEx will own this, but I can remain DRI for now. We will likely want to split this and have something for 8.8 anyway, but we're reviewing that this week as part of the backup workshop.
